website/static/js/filepage/index.js

```diff
@@ -363,6 +363,64 @@ var FileViewPage = {
         }, 1000);
 
         self.mfrIframeParent = $('#mfrIframeParent');
+        function goToRevisions(e){
+          var editable = self.editor && self.editor.selected;
+          var viewable = self.mfrIframeParent.is(':visible');
+          if (editable || viewable){
```

Comment:
Please do change it. It's entirely trivial from a performance view either way, but in this case the code is entirely redundant.